pkg/test/resourcefixture/testdata/basic/bigtable/v1beta1/bigtableappprofile/_http.log

```diff
@@ -383,10 +359,25 @@ X-Frame-Options: SAMEORIGIN
 X-Xss-Protection: 0
 
 {
+  "done": true,
```

Comment:
This diff is in the PATCH call for appProfiles, so I think we should update the mock to match it. However, this can be handled in a separate PR.